.browserslistrc

```diff
@@ -1 +1 @@
-chrome >= 80, firefox >= 78
+chrome >= 89, firefox >= 89
```

Comment:
Looks like these versions are below the `OUTDATED_BROWSER_VERSIONS` we have configured, so this looks good :+1: